packages/rolldown/src/cli/commands/bundle.ts

```diff
@@ -14,13 +19,133 @@ export async function bundle(configExport: RolldownConfigExport) {
 }
 
 async function bundleInner(options: RolldownOptions) {
-  const start = performance.now()
+  const dir = options.output?.dir ?? 'dist'
```

Comment:
The default for `dir` is held by the Rust side.
https://github.com/rolldown/rolldown/blob/667c7df335e998cd52275da2225827b0d7e17201/crates/rolldown/src/utils/normalize_options.rs#L83

I think we need a mechanism to share `dir` defaults between JS side and Rust side.
If we change this value on either side, it may not work.

Reply:
For non-gziped size, you should able to get them in the returned value of `write` or `generate`. However, this might still slow down the performance potentially if we didn't do the optimization in #696. 

I suggest we now only print non-gized sizes. It should solves
- the performance problem of large project
- not knowing `default for dir` problem